src/main/java/com/google/cloud/resourcemanager/OrganizationInfo.java

```diff
@@ -0,0 +1,312 @@
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.core.ApiFunction;
+import com.google.api.services.cloudresourcemanager.model.Organization;
+import com.google.cloud.StringEnumType;
+import com.google.cloud.StringEnumValue;
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.Objects;
+import org.threeten.bp.Instant;
+import org.threeten.bp.ZoneOffset;
+import org.threeten.bp.format.DateTimeFormatter;
+
+/** A Google Cloud Resource Manager organization metadata object. */
+public class OrganizationInfo implements Serializable {
+
+  public static final DateTimeFormatter DATE_TIME_FORMATTER =
+      DateTimeFormatter.ISO_DATE_TIME.withZone(ZoneOffset.UTC);
+  private Long creationTime;
+  private String displayName;
+  private State lifecycleState;
+  private String name;
+  private OrganizationOwner owner;
+
+  public static final class State extends StringEnumValue {
+
+    private static final ApiFunction<String, State> CONSTRUCTOR =
+        new ApiFunction<String, State>() {
+          @Override
+          public State apply(String constant) {
+            return new State(constant);
+          }
+        };
+    private static final StringEnumType<State> type = new StringEnumType(State.class, CONSTRUCTOR);
+
+    /** Useful for distinguishing unset values. */
+    public static final State LIFECYCLE_STATE_UNSPECIFIED =
+        type.createAndRegister("LIFECYCLE_STATE_UNSPECIFIED");
+
+    public static final State ACTIVE = type.createAndRegister("ACTIVE");
+
+    /**
+     * Represents the state of organization that has been marked for deletion from GCP by the user
+     * or by the system (Google Cloud Platform).
+     */
+    public static final State DELETE_REQUESTED = type.createAndRegister("DELETE_REQUESTED");
+
+    private State(String constant) {
+      super(constant);
+    }
+
+    /**
+     * Returns the State for the given String constant, and throw an exception if the constant is
+     * not recognized.
+     */
+    public static State valueOfStrict(String constant) {
+      return type.valueOfStrict(constant);
+    }
+
+    /** Returns the State for the given String constant, and allow unrecognized values. */
+    public static State valueOf(String constant) {
+      return type.valueOf(constant);
+    }
+
+    /** Returns the known values for State. */
+    public static State[] values() {
+      return type.values();
+    }
+  }
+
+  public static final class OrganizationOwner implements Serializable {
```

Comment:
outer class

Reply:
no it should be inner class because its only used at one place just need to remove access specifier `public`